libraries/botbuilder/tests/botFrameworkAdapter.test.js

```diff
@@ -351,9 +351,11 @@ describe(`BotFrameworkAdapter`, function () {
 
         it('ConnectorClient should use httpClient from clientOptions', async () => {
             let sendRequestCalled = false;
+            const outgoingMessageLocale = JSON.parse(JSON.stringify(outgoingMessage));
```

Comment:
I had to make a change to this test because instead of an activity with no locale coming back as:

```js
{
  ...
  text: 'blah'
  ...
}
```

...it would come back as:

```js
{
  ...
  text: 'blah',
  locale: undefined
  ...
}
```

I'm a *little* worried that `activity.locale = reference.locale` (in the above changes) could be a rare breaking change (should really only happen with something like `assert.deepEqual()` or `if (activity.locale !== undefined)`(?). This could be fixed by using something more like:

```js
if (!reference.locale) { activity.locale = reference.locale; }
```

...but I figured the very low risk made the readability okay. Let me know if you feel otherwise.